Coalition Scientist Missions (#6798)
endless-sky/endless-sky: data/coalition/coalition missions.txt

			`   This would mean to travel to Hai territory. As there is no data in human space that is public.`
			choice
				`	"I will do everything for science."`
				`	"Sorry but that's a too long way for me."`
					decline
			`	"Ah thanks to you." He wishes you best luck and goes away.`
				accept
	on stopover Hai-home
		conversation
			`When you landed on Hai Home you ask some people around for an historic library where you could find the info the Kimek is searching for. To your pleasure it is not taking long for gathering some data and you are ready to get back again.`
	on complete
		conversation
			`After your landing you directly go to the Kimek and give hime the data device. Happy that he got what he was searching for he says "Thanks to you human. I really appretiate your help. Carefully we will scan each detail of this info. Maybe we will need some more another time. Hope i can count on your help then." After he said that he is going away.`

		"coalition jobs" += 1
		payment 100000

> 		event "coalition: interest raises" 12

Reviewer: It's not necessary to do what I'll explain, but just in case you didn't know and want to make use of it, you can set intervals for these timer events.

For instance, if you wanted the timer to be somewhere between 12 and 20 days, you'd just add a 20 after the 12 (so the line would be `event "coalition: interest raises" 12 20`.

Don't need to do that if you don't want to, just pointing out you can if you didn't already know you could.

> 

Reviewer: It's custom to have the events given by a mission also defined separate, normally right below the mission in question. I don't really know why it's done, but it's how it is for most events, so just to keep it consistent.
```suggestion

event "coalition: interest raises"

```

mission "Coalition: Hunger for Knowledge"
	landing
	name "Hunger for Knowledge"
	description "A Heliarch Scientists team wants to know more about the Wormhole connecting Human and Hai-Space"
	to offer

> 		has "Coalition: Interesting Connections: done"

Reviewer: The mission already checks for the timer event (interest raises), so you don't need to check for this mission being done, as it has to be done for you to get the event in the first place.
```suggestion

```

		has "event: coalition: interest raises"
	source
		planet "Ring of Wisdom"
	waypoint "Ultima Thule"
	waypoint "Waypoint"
	destination
		planet "Ring of Wisdom"
	on offer
		conversation

> 			`Directly after your landing the same Kimek from the last time is approaching you. "Hey Captain, another job for you i have."`

Reviewer: ```suggestion
			`Directly after your landing the same Kimek from the last time is approaching you. "Hey Captain, another job for you I have."`
```

			`    You turn around "What is it this time?" He responds "Both sides of the wormhole you could scan them for me?"`
			choice
				`	"Yeah i got some time."`
				`   "What do you think you will find?"`
					goto purpose
				`	"I'm not looking for work right now"`
					decline
			`	"Very well take this small device it will do the scans." `
				accept
			label purpose
			`    "No wormholes we have in our region of space. That's why we would like to increase our knowledge about them. Furthermore we hope we get more information about the structure of space itself and it brings us nearer to creatng jump drives we hope." You respond: "I would be curoius for that."`
			`    "Taking time this will but sometime in the future we will get out of this cages and see the whole galaxy. So will you do it or not?"`
			choice

> 				`	"Fine i will do it."`

Reviewer: ```suggestion
				`	"Fine I will do it."`
```

> 				`	"Sorry but i have no time for something like that."`

Reviewer: ```suggestion
				`	"Sorry but I have no time for something like that."`
```

					decline
			`   He hands you a small device. "Attach this to your ships scanners i will wait for your results." As you look at it closer it looks like somekind of inverted satellite dish and you have no idea what it could be used for. But not asking further question you go to your ship.`
			    accept
	on complete
		conversation
			`With the data of the scanners you go to the Kimek. When you find him you hand the data over to him.`
			`    He takes the data device and thanks you. "It will take some time to evaluate the data but when i got something i will tell you."`
			choice
				`	"I am looking forward hearing the results"`
				`   "Well see you."`
		"coalition jobs" += 1
		payment 200000
		event "coalation: wormhole evaluated" 17

> 

Reviewer: Same thing with events.
```suggestion

event "coalition: wormhole evaluated"

```

mission "Coalition: Last Info"
	landing
	name "Last Info"
	description ""
	to offer
		random < 80
		has "Coalition: Hunger for Knowledge: done"
		has "event: coalation: wormhole evaluated"
	source
		planet "Ring of Wisdom"
	on offer
		conversation
			`When you enter the spaceport you see the Kimek from the science stuff taking to another Kimek. As he recognizes you he walks towards you.`

> 			`    "Hello Captain thanks again for the help. Our horizon of knowledge was increased by you. That said we are far away from knowing all about them but we can make further studies based on that data. In the name of our Group I thank you. If we ever need help again i will contact you."`

Reviewer: ```suggestion
			`    "Hello Captain thanks again for the help. Our horizon of knowledge was increased by you. That said we are far away from knowing all about them but we can make further studies based on that data. In the name of our Group I thank you. If we ever need help again I will contact you."`
```

				`	"This is getting to dangerous for me, sorry."`
					decline
			`	"Good we have a second one ready in ten minutes. Please wait so long."`
			`	Ten minutes later the probe is installed you go for another trip to the supernova.`
				accept
		outfit "Heat Shield"
	on waypoint
		dialog "After you entered the system you deploy the probe. Hopefully it will work this time."
	on fail
		outfit "Heat Shield" -1
	on complete
		outfit "Heat Shield" -1

event "Questions novad"
	system Questions
		pos -1210.05 673.394
		government Uninhabited
		hazard "Pre Supernova" 1
		remove object "S1"
			sprite star/wr1
			period 12.6101
		remove object "S2"
			sprite star/k5-old
			period 12.6101
		add object
			sprite star/nova
			period 1000
		add object
			sprite star/nova-core
			period 1000

mission "Coalition: Supernova Explorers: Conclusion"
	landing
	name "Supernova research V"
	description ""
	to offer
		has "Coalition: Supernova Explorers: Fixing Problems: done"
	source
		planet "Bright Echo"
	on offer
		conversation
			`When you got to the Lab the Kimek is shouting to you. "Come here we are getting first pictures of the nova."`
			scene scene/supernova

> 			`	All of you are looking at the impressive pictures for some while. After the Nova ended the Kimek is talking to you again. "It was ice to work with you. I hope we will meet again in the future."`

Reviewer 2: Possible change?:
```suggestion
			`	All of you are looking at the impressive pictures for some while. After the Nova ends the Kimek is talking to you again. "It was nice to work with you. I hope we will meet again in the future."`
```
